config_examples/dev.env.example

```diff
@@ -0,0 +1,10 @@
+# please define a data-directory on your host machine
+# this will map to /data/wis2box on the wis2box-container
+WIS2BOX_HOST_DATADIR=your/data/directory
+
+# Optional
+# Environment variable overrides
+WIS2BOX_LOGGING_LOGLEVEL=WARNING
+WIS2BOX_DATA_RETENTION_DAYS=30
+
+WIS2BOX_DATADIR_DATA_MAPPINGS=/data/wis2box/data-mappings.yml
```

Comment:
This needs to go in the Docker default.env file.

Reply:
Yes they are originally defined in docker/default.env, by redefining them in dev.env the user overrides them. this is how the environment variables in docker-compose currently are implemented. Could be done differently, these instructions are to guide the user to work with how it is done now...

Reply:
could you check my proposal for baseline data-mappings ? I was wondering, the bufr-example is not actually synop specific is it ?